tests/log/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![no_std]
+use sdk::Val;
+use stellar_contract_sdk as sdk;
+
+#[no_mangle]
+pub fn log(v: Val) -> Val {
+    sdk::log_value(v);
+    Val::from_void()
+}
+
+#[cfg(test)]
+mod test {
+    use super::log;
+    use sdk::{Symbol, Val};
+    use stellar_contract_sdk as sdk;
+
+    #[test]
+    fn test_log() {
+        log(Val::from_i64(7));
+        log(Val::from_i64(-7));
+        log(Symbol::from_str("asdf").into());
```

Comment:
This test will output the following with this PR and the fix from #34:
```
MemHost::log_value: Val { u63: 7 }
MemHost::log_value: Val { object: Object { type: 4, idx: 0, 0: 0 } }
MemHost::log_value: Val { symbol: Symbol { str: "asdf" } }
```
The second value is the `I64` object, which still doesn't render wonderfully, but this is all a step in the right direction.